content/blog/discover-the-power-of-data-center-monitoring-using-redfish-telemetry-and-cloud-native-tooling-part-2-streaming.md

```diff
@@ -0,0 +1,42 @@
+---
+title: "Discover the power of data center monitoring using Redfish telemetry and
+  cloud-native tooling: Part 2 - streaming"
+date: 2023-12-01T21:50:51.315Z
+author: Naveen Gupta
+authorimage: /img/photo-jpg-ng.jpg
+thumbnailimage: ""
+disable: false
+tags:
+  - ilo-restful-api
+  - Grafana
+  - Telegraf
+  - Telemetry
+  - InfluxDB
+  - Redfish
+  - monitoring
+---
+<style>
+li {
+   font-size: 27px;
+   line-height: 33px;
+   max-width: none;
+}
+</style>
+
+Unlock the full potential of data center monitoring with Redfish telemetry streaming and cutting-edge cloud-native tools. The second part of our data center monitoring whitepaper series delves into the world of streaming, showcasing the efficiency and scalability of HPE iLO's Redfish telemetry streaming using event subscription.
+
+# A Holistic Approach
+
+Embark on a comprehensive journey through my latest technical whitepaper, **"[Data Center Monitoring using Redfish Telemetry and Cloud-Native Tooling: Part 2 – Streaming](https://www.hpe.com/psnow/doc/a50009739enw)"**. This guide not only introduces you to the intricacies of Redfish telemetry streaming but also demonstrates the seamless integration with open-source cloud-native tools such as Telegraf, InfluxDB, and Grafana. The telemetry data source, HPE iLO, provides a wealth of performance metrics through the Redfish interface, and event subscription ensures a continuous flow of metric reports from the telemetry service.
+
+# Key Highlights from the Whitepaper
+
```

Comment:
Before you jump into the bullets, you might want to just add a short paragraph that reads something like this:

"In my whitepaper, I provide you with a deep understanding of:"

Then, take off "Understanding" at the beginning of the first bullet.

Reply:
Fixed.